### PR TITLE
fix: fixing possible default z-index problems

### PR DESCRIPTION
**Pull request by marvinLaubenstein**
⚠️ Please check if this is wanted like this before merging. 

current status:
Bildschirmfoto 2022-06-02 um 21 03 25
with changed z-index by variable:
Bildschirmfoto 2022-06-02 um 22 00 35

I noticed a problem with the fixed z-indexes (compare screenshot). I think it can definitely be useful for the user of the component if he can set the z-index variably. 

In the variables I have set the old values as default these change nothing and could also be 0. But someone has certainly thought something with the values.

### CODE REVIEW

File: packages/components/src/components/date-picker/date-picker.css [hunk 1]
```diff
@@ -100,6 +102,7 @@ duet-date-picker .duet-date__toggle:active scale-icon-content-calendar {
 }
 
 .scale-date-picker {
+  --z-index-label: 10;
```
Comment (nowseemee) [inline packages/components/src/components/date-picker/date-picker.css:105]: should we add this to the top of the file, or is there a reason you've inserted a new variable into this position? thanks :)

Comment (acstll) [reply packages/components/src/components/date-picker/date-picker.css:105]: @marvinLaubenstein I agree with Daniel this line should be moved up, to place it inside `:host`, around line 34. Other than that, this looks fine!

**Review by acstll — CHANGES_REQUESTED** [review]
Left a comment, I think the `--z-index-label` declaration should be placed in the :host. Or is there a reason not to do it?

**Comment by nowseemee** [comment]
@acstll do you approve?

**Comment by acstll** [comment]
Will also take a look @nowseemee — sorry I missed the mention (I need to fix my email/notification/etc.) 😅

**Review by acstll — APPROVED** [review]

**Comment by acstll** [comment]
done… @marvinLaubenstein LGTM

**Pull request merged**